docs/client-metrics.md

```diff
@@ -196,7 +196,8 @@ The event stream is a log of events and the time they occurred while the user is
 
 #### signup
 * tooltip.mailcheck-suggested - an email address correction was suggested
-* tooltip.mailcheck-used - an email address correction was chosen by the user
+* tooltip.mailcheck-used - an email address correction was chosen by the user in the tooltip
+* tooltip.mailcheck-useful - an email address correction was used by the user during sign up form submit
```

Comment:
Can you add a bit of meta info to this event description that it covers both the tooltip and manually changing the address (my guess at what the new event is for).